scripts/pretraining/pretraining_utils.py

```diff
@@ -451,15 +451,16 @@ class ElectraMasker(HybridBlock):
                                           'masked_positions',
                                           'masked_weights'])
 
-    def __init__(self, tokenizer, max_seq_length, mask_prob,
-                 proposal_distribution=1.0):
+    def __init__(self, tokenizer, max_seq_length, mask_prob=0.15,
+                 proposal_distribution=1.0, replace_prob=0.85):
```

Comment:
Add some documentations about the `tokenizer`, `max_seq_length`, `mask_prob`, ...